### ISSUE TITLE
hello,Can you compile the c + + source code into the android code

### ISSUE THREAD

**Issue by xuetenglong** [mcodegeeks/RippleEffect#1]
please ,thanks you

**Comment by xuetenglong** [comment]
Because I don't understand c + +, I hope you can help me

**Comment by ry-narola** [comment]
yes I also had the same issue ..how we can run it in Android Studio?

**Comment by brogan89** [comment]
Have you guys had any luck with this code in Java? - I know a little bit of C++ and I will attempt to convert the code to Java. But if you have any progress so far I'd love to see :)
Once I convert it I will post the Java equivalent on my github and link here. 
I will be using Android Studio and LibGDX
